lms/djangoapps/etc_api/views.py

```diff
@@ -20,26 +19,28 @@
 
 log = logging.getLogger(__name__)
 
+
 def string_to_boolean(string):
     return bool(string) and str(string).lower() == 'true'
 
 
-class CreateUserAccountWithoutPasswordView(APIView):
+class CreateUserAccountView(APIView):
     authentication_classes = (OAuth2AuthenticationAllowInactiveUser,)
     permission_classes = ApiKeyHeaderPermission,
 
     def post(self, request):
         """
-        Create user account without password
+        Create user account
 
         Creates a user using mail, login and also name and surname.
-        Sets a random password and sends a user a message to change it.
+        Sets sent in request password and sends a user a message to change it.
         """
         data = request.data
         data['honor_code'] = "True"
         data['terms_of_service'] = "True"
         email = request.data.get('email')
         username = request.data.get('username')
+        password = request.data.get('password')
```

Comment:
Please clarify do we need to handle password validation in case it fails to send a meaningful response.

Reply:
Add validation of "predefine" password